srv/salt/_modules/osd.py

```diff
@@ -898,8 +898,10 @@ def _bluestore_partitions(self):
                     log.warning(("WAL size is unsupported for same device of "
                                  "{}".format(self.osd.device)))
                 else:
-                    # Create wal of wal_size on wal device
-                    self.create(self.osd.wal, [('wal', self.osd.wal_size)])
+                    log.info(("WAL will reside on same device {} as db - "
```

Comment:
Perhaps I missed something but I don't see the difference in handling following two cases: 
1) wal == db 
2) wal != db
IMO both are getting here but there is no wal creation for case 2)

Reply:
Right, the wal would not be created in either case, but the reason is different for the admin.  The first message is that ceph-disk does not support specifying a wal on the same device as the OSD.  The second is that it's unnecessary.